MANIFEST

```diff
@@ -521,8 +528,6 @@
 ./src/hdf5.lnt        _DO_NOT_DISTRIBUTE_
 ./src/hdf5-win.lnt      _DO_NOT_DISTRIBUTE_
 ./src/hdf5-lin.lnt      _DO_NOT_DISTRIBUTE_
-./src/hlog.c
-./src/hlog.h
```

Comment:
The logging layer uses POSIX things that are not present on Windows and most of it looked like printf debugging, so I ripped it out.

Reply:
Here are my belated comments. Apparently I clicked "Start a review" and that queues comments to submit in a batch.

Reply:
We can revisit this later. I'd be excited about adding a logging layer to HDF5. I'm not so excited about a logging system that is non-portable and implemented ina scattershot manner.